tests/virtualization/external/prepare.pm

```diff
@@ -15,19 +15,24 @@ use version_utils;
 
 sub run {
     my ($self) = @_;
-    $self->select_serial_terminal;
-
+    select_console("root-console");
     script_run("SUSEConnect -r " . get_var('SCC_REGCODE'), timeout => 420);
-
     assert_script_run "rm /etc/zypp/repos.d/SUSE_Maintenance* || true";
     assert_script_run "rm /etc/zypp/repos.d/TEST* || true";
     zypper_call '-t in nmap iputils bind-utils', exitcode => [0, 102, 103, 106];
 
     # Fill the current pairs of hostname & address into /etc/hosts file
-    assert_script_run "echo \"\$(dig +short $virt_autotest::common::guests{$_}->{ip}) $_ # virtualization\" >> /etc/hosts" foreach (keys %virt_autotest::common::guests);
+    if (get_var("REGRESSION", '') =~ /vmware/) {
+        foreach my $guest (keys %virt_autotest::common::guests) {
+            my $ip = script_output(qq(ssh -o StrictHostKeyChecking=no root\@esxi7.qa.suse.cz "vim-cmd vmsvc/get.guest \\`vim-cmd vmsvc/getallvms | grep -w $guest|cut -d ' ' -f1\\`|grep -A 1 hostName|grep ipAddress|cut -d '\\"' -f2"));
+            record_info("$ip");
```

Comment:
@tonyyuan1  Can we make this record_info more informative? eg sth like "Found guest $guest ip : $ip"?

Reply:
yes

Reply:
@tonyyuan1  Sorry I forgot this one and merged it. There is no easy way to reopen the PR unless revert then submit PR again. Can we have it merged and you open a new small PR for this comment related changes?

Reply:
@alice-suse No problem. I will do that.